client/services/FeedbackGeneratorService.js

```diff
@@ -415,6 +415,20 @@ tie.factory('FeedbackGeneratorService', [
             'Please keep your code within the existing predefined functions',
             '-- we cannot process code in the global scope.'
           ].join(' '));
+        } else if (prereqCheckFailure.hasWrongLanguage()) {
+          WRONG_LANGUAGE_ERRORS.python.forEach(function(error) {
+            if (error.errorName === prereqCheckFailure.getWrongLangKey()) {
+              error.feedbackParagraphs.forEach(function(json) {
+                if (json._type === 'text') {
```

Comment:
Oh wait, I think I get it now -- this is actually reading from a constant? If so, I think referring to .type is fine, sorry.